src/gui/guiFormSpecMenu.cpp

```diff
@@ -779,6 +780,48 @@ void GUIFormSpecMenu::parseImage(parserData* data, const std::string &element)
 	errorstream<< "Invalid image element(" << parts.size() << "): '" << element << "'"  << std::endl;
 }
 
+void GUIFormSpecMenu::parseAnimatedImage(parserData *data, const std::string &element)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	if (parts.size() != 3 &&
+			(parts.size() < 3 || m_formspec_version <= FORMSPEC_API_VERSION)) {
+		errorstream << "Invalid animated image element(" << parts.size() << "): '" << element << "'"  << std::endl;
```

Comment:
Line is longer than 80 characters. Please break it.

Reply:
You're right that the line is too long, and I've gone in and broken it. I think the result is harder to read though, I'd like your opinion on how I did the break and if you see a better spot to do it.

Reply:
For two lines it's fine to break it like this. If it were longer, passing the template to `snprintf` would be a good solution.
Optimally binary operators would be placed to the end of the previous line. I think both is fine, so I'm not going to enforce that.